### PR TITLE
Readme + Contributing + Gemspec updates

### PR DESCRIPTION
**Pull request by tas50**
1 per commit

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,155 +1,161 @@
...
-
-The [ticket tracker](http://tickets.opscode.com) is the most important documentation for the code base. It provides significant historical information,
-such as:
+Once you a pull request, the **Chef Engineering Team** or **Chef Core Committers** will review your code
+  and respond to you with any feedback they might have. The process at this point is as follows:
 
-* Which release a bug fix is included in
-* Discussion regarding the design and merits of features
-* Error output to aid in finding similar bugs
+1. 2 thumbs-ups are required from the **Chef Engineering Team** or **Chef Core Committers** for all merges.
```
Comment (chefsalim) [inline CONTRIBUTING.md]: Change 2 to "two"

**Comment by chefsalim** [comment]
👍 with minor comment

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,155 +1,161 @@
...
-*   Sign our contributor agreement (CLA) [
-online](https://secure.echosign.com/public/hostedForm?formid=PJIF5694K6L)
-    (keep reading if you're contributing on behalf of your employer)
-* Create a ticket for your change on the [bug tracker](http://tickets.opscode.com)
-* Link to your patch as a rebased git branch or pull request from the ticket
-* Resolve the ticket as fixed
+## Contribution Process
 
-We regularly review contributions and will get back to you if we have any suggestions or concerns.
+We have a 3 step process that utilizes **Github Issues**:
```
Comment (chefsalim) [inline CONTRIBUTING.md]: Change "3" to "three"

**Comment by tas50** [comment]
@chefsalim: Now that 1.0 is released can we merge this?

**Comment by tas50** [comment]
@chef/client-core This is ready for review

File: README.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+To build and install the plugin, run:
+
+```shell
+    rake install
+```
+
+## Configuration:
+
+If push server is running on the same host as the OPC, then no reconfiguration is required on the Chef workstation.
```
Comment (thommay) [inline README.md]: `OPC` -> `Chef Server`.

**Comment by tas50** [comment]
@thommay Readme has been updated + some other testing + lint + gemspec updates

**Comment by btm** [comment]
let's fork rubocop and name it styleshed.

:+1:

**Comment by thommay** [comment]
👍

**Pull request merged**